PR: https://github.com/odoo/odoo/pull/178200

From: f10bd8ddcf99c9247dca417fefbc260454d74ea1
From: Gorash
Date: 2024-10-15 09:33:02

Breaking data model changes scores: del:4 + add:0, change matches:
-    _inherit = "project.project"
-    _inherit = 'project.project.stage'
-    _inherit = "project.task"
-    _inherit = "project.task.type"

Total Changes: 8

[IMP] *: adapt model class names to correspond to model names (apply script)

Part-of: odoo/odoo#178200
Related: odoo/enterprise#69762
Related: odoo/documentation#11115
Related: odoo/upgrade#6577
Related: odoo/design-themes#988
Related: odoo/upgrade-util#143

================================= pseudo patch: =================================

--- a/addons/project_sms/models/project_project.py
+++ b/addons/project_sms/models/project_project.py
@@ -5,7 +5,7 @@ from odoo import api, models
 
 
 class ProjectProject(models.Model):
-    _inherit = "project.project"
+    _inherit = ["project.project"]
 
     def _send_sms(self):
         for project in self:

--- a/addons/project_sms/models/project_stage.py
+++ b/addons/project_sms/models/project_stage.py
@@ -5,7 +5,7 @@ from odoo import fields, models
 
 
 class ProjectProjectStage(models.Model):
-    _inherit = 'project.project.stage'
+    _inherit = ['project.project.stage']
 
     sms_template_id = fields.Many2one('sms.template', string="SMS Template",
         domain=[('model', '=', 'project.project')],

--- a/addons/project_sms/models/project_task.py
+++ b/addons/project_sms/models/project_task.py
@@ -5,7 +5,7 @@ from odoo import api, models
 
 
 class ProjectTask(models.Model):
-    _inherit = "project.task"
+    _inherit = ["project.task"]
 
     def _send_sms(self):
         for task in self:

--- a/addons/project_sms/models/project_task_type.py
+++ b/addons/project_sms/models/project_task_type.py
@@ -5,7 +5,7 @@ from odoo import fields, models
 
 
 class ProjectTaskType(models.Model):
-    _inherit = "project.task.type"
+    _inherit = ["project.task.type"]
 
     sms_template_id = fields.Many2one('sms.template', string="SMS Template",
         domain=[('model', '=', 'project.task')],
